packages/sdk/src/swap/types.ts

```diff
@@ -90,7 +104,10 @@ export type SwapStatusResponse = CommonStatusFields &
         swapExecutedBlockIndex: string;
         egressAmount: string;
         egressScheduledAt: number;
-        egressCompletedAt: number;
```

Comment:
renaming this from `egressCompleted` to `broadcastSuccess` to match the event naming